app/views/ProjectsView/columnDefs.ts

```diff
@@ -16,7 +16,7 @@ const readOnlyColDefs = [
   },
   {
     headerName: 'Number of reports',
-    valueGetter: ({ data }) => data?.reports?.length,
+    valueGetter: ({ data }) => Number(data?.reportCount),
```

Comment:
not sure why the API comes back as string representation, but I'm not going to question it